spec/models/automation_request_spec.rb

```diff
@@ -88,6 +88,61 @@
     end
   end
 
+  context ".create_from_scheduled_task" do
+    let(:admin) { FactoryGirl.create(:user_miq_request_approver) }
+
+    it "with prescheduled task" do
+      ar = AutomationRequest.create_from_scheduled_task(admin, @uri_parts, @parameters)
+      expect(ar).to be_kind_of(AutomationRequest)
+      expect(ar).to eq(AutomationRequest.first)
```

Comment:
What are you trying to test here?  If you're trying to test that a record is created, you could do:
```
expect{ar = AutomationRequest.create_from_scheduled_task(admin, @uri_parts, @parameters)}.to change{AutomationRequest.count}.from(0).to(1)
```